xarray/tests/test_duck_array_ops.py

```diff
@@ -301,3 +308,17 @@ def test_argmin_max_error():
     da = construct_dataarray(2, np.bool_, contains_nan=True, dask=False)
     with pytest.raises(ValueError):
         da.argmin(dim='y')
+
+
+@requires_dask
+@pytest.mark.parametrize('axis', [0, -1])
+@pytest.mark.parametrize('window', [3, 8, 11])
+def test_dask_rolling(axis, window):
```

Comment:
It looks the test fails if the environment does not have dask.
It says 
```
../../../miniconda/envs/test_env/lib/python2.7/site-packages/_pytest/python.py:798: in parametrize
    self.function, name, arg))
E   ValueError: <function test_dask_rolling at 0x7f963326d050> uses no argument 'window'
```
Can we use both `@requires_dask` and `@pytest.mark.parametrize`?

Reply:
@fujiisoup - I've found that this doesn't always work. In some cases, I've reverted to using `importorskip` inside test functions.

Reply:
I think it might work if you make `@requires_dask` the inner-most decorator. But that's just a guess...

Reply:
Thanks. 

I noticed that moving this one to the inner-most decorator does not solve this issue.
I decided to use `pytest.skipif(has_dask, ...)` instead.